sql/src/test/java/org/apache/druid/sql/calcite/DecoupledTestConfig.java

```diff
@@ -86,10 +84,8 @@ enum QuidemTestCaseReason
      * New scans / etc.
      */
     DEFINETLY_WORSE_PLAN,
-    /**
-     * A new {@link FinalizingFieldAccessPostAggregator} appeared in the plan.
-     */
-    FINALIZING_FIELD_ACCESS;
+    EQUIV_PLAN_EXTRA_COLUMNS,
+    EQUIV_PLAN_CAST_MATERIALIZED_EARLIER;
```

Comment:
I think I know what `EXTRA_COLUMNS` means, but `CAST_MATERIALIZED_EARLIER` I'm unclear on.  Can you add the comments for these like you have on the others?